technical-reports/resolver/resolution-example.md

```diff
@@ -1,199 +0,0 @@
-# Resolver Resolution Example
```

Comment:
This example was kept and just moved as the primary example under `resolution-logic.md`